packages/docs-examples/components/select/select-footer/select-footer-example.html

```diff
@@ -52,12 +52,17 @@
         <kbq-option [value]="'SelectedItem9'">SelectedItem9</kbq-option>
         <kbq-option [value]="'SelectedItem10'">SelectedItem10</kbq-option>
         <kbq-select-footer>
-            <div class="kbq-select__no-options-message kbq-caption">Подсказка о чем-нибудь ⌥+⌘+F</div>
+            <div
+                style="padding: 0; display: block; height: unset"
```

Comment:
По макету подсказка должна быть серой
```suggestion
                style=" color: var(--kbq-foreground-text-less-contrast);"
```

Reply:
парни давайте не будем трогать стили, нужно как можно быстрей перевести все на стендалоун

писать `style` в темплейте не очень хорошая практика, а тащить еще css переменные уж темболее

Reply:
@artembelik согласен, но раз уж начали менять стили, то ничего сломаться так же не должно
@sk8knut цвет текста поправить нужно , либо вернуть как было